parsers/IL.py

```diff
@@ -130,14 +140,19 @@ def fetch_consumption(
         raise NotImplementedError("This parser is not yet able to parse past dates")
 
     data = fetch_all()
-    consumption = [float(item) for item in data]
-
-    # all mapped to unknown as there is no available breakdown
+    consumption = data.get("Production")
+    if not consumption:
+        raise ValueError("Could not parse Noga consumption")
+    date = data.get("PeakReserveDate")
+    time = data.get("PeakReserveTime")
+    date_time = arrow.get(f"{date} {time}", "DD/MM/YYYY HH:mm").datetime
```

Comment:
Same here as above and it looks like this could be turned into a function used both above and here.